Jenkinsfile

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env groovy
+
+import groovy.transform.Field
+
+@Field
+def TAG = ""
+
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+  }
+
+  triggers {
+    cron(getDailyCronString())
+  }
+
+  stages {
+    parallel {
+
+      # For available Docker tags for Kubernetes node images see:
+      #     https://github.com/kubernetes-sigs/kind/releases
+
+      stage('Helm Test on Kubernetes v1.18.2') {
+        environment {
+          KUBE_NODE_TAG   = "v1.18.2@sha256:7b27a6d0f2517ff88ba444025beae41491b016bc6af573ba467b70c5e8e0d85f"
+          KUBECTL_VERSION = "1.18.2"
+          CLUSTER_ID      = "1.18.2"
+          GIT_REF = sh(returnStdout: true, script: 'echo $(< VERSION)-$(git rev-parse --short HEAD) | tr -d "\n"')
+        }
+        steps {
+          sh 'cd ci && ./jenkins_build.sh'
+          script {
+            KUBE_NODE_TAG   = "${env.KUBE_NODE_TAG}"
+            KUBECTL_VERSION = "${env.KUBECTL_VERSION}"
+            CLUSTER_ID      = "${env.CLUSTER_ID}"
+            GIT_REF         = "${env.GIT_REF}"
```

Comment:
I'm leaning a little towards not explicitly defining `environment` for these since you're just directly using these variables right afterwards in a single spot each time.